packages/core/src/client.ts

```diff
@@ -26,17 +26,17 @@ import core from './component'
 type TxHander = (tx: Tx) => void
 
 export interface Client extends Storage {
-  isDerived<T extends Obj> (_class: Ref<Class<T>>, from: Ref<Class<T>>): boolean
+  isDerived: <T extends Obj>(_class: Ref<Class<T>>, from: Ref<Class<T>>) => boolean
```

Comment:
Why? :)

Reply:
This is standard rule and eslint with standard format it automatically this way.

Reply:
This looks really strange because I'm pretty sure `ts-standard` do not force such a rule. From my side I do not really like to have interface definitions like this. Can we explore why `ts-standard` is different?

Reply:
ts-standard for me shows same output added to similar comment in previous comment.